pom.xml

```diff
@@ -156,7 +156,11 @@
         <artifactId>moshi</artifactId>
         <version>${moshi.version}</version>
       </dependency>
-
+      <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>jsr305</artifactId>
+        <version>3.0.2</version>
```

Comment:
Putting it here forces all submodule consumers to use it as provided. The same would be true if you put 'test' then only submodules could use it as a test artifact. As a rule I generally put only the dependency pointer in the root and then let the submodules decide how they want to actually use it.

But it really doesn't matter...